page/partials/key-reading.ejs

```diff
@@ -2,22 +2,22 @@
     <div class="container">
         <div class="row">
             <div class="col-sm-12">
-                <h2 class="section-title">Key reading</h3>
+                <h2 class="section-title">Key reading</h2>
             </div>
             <div class="col-md-offset-1 col-md-5">
                 <ul class="separator compact-list">
                     <% for(var i = 0; i < 3; i++) { %>
-                        <li><i class="material-icons"><%- links[i].downloadable? "file_upload" : "link" %> </i><a href="<%- links[i].href %>" target="_blank" ><%- links[i].link %></a></li>
+                        <li><i class="material-icons"><%- links[i].downloadable? "file_download" : "link" %> </i><a href="<%- links[i].href %>" target="_blank" ><%- links[i].link %></a></li>
```

Comment:
can u do global grep for `file_upload` and replace it everywhere? we use it in react as well. And it should be download everywhere.